Solutions/Proofpoint On demand(POD) Email Security/Parsers/ProofpointPOD.yaml

```diff
@@ -314,4 +379,245 @@ FunctionQuery: |
+                    FilterVerifiedRcpts,
+                    Guid,
+                    EnvelopeRcptsHashed,
+                    EnvelopeFromHashed,
+                    MsgNormalizedHeaderReturnPathHashed,
+                    MsgNormalizedHeaderXMailer,
+                    MsgNormalizedHeaderReturnPath,
+                    MsgHeaderXMailer,
+                    MsgHeaderReturnPathHashed,
+                    MsgHeaderReturnPath,
+                    FilterModulesSpamCharsets,
+                    FilterModulesDmarcAlignment,
+                    MsgNormalizedHeaderXOriginatingIp,
+                    MsgHeaderXOriginatingIp,
+                    MsgNormalizedHeaderReplyTo,
+                    MsgNormalizedHeaderReplyToHashed,
+                    MsgHeaderReplyToHashed,
+                    MsgHeaderReplyTo,
+                    FilterModulesUrldefenseCountsNoRewriteIsEmail,
+                    FilterModulesPdrV2Rscore,
+                    FilterOrigGuid,
+                    MsgNormalizedHeaderCc,
+                    MsgNormalizedHeaderCcHashed,
+                    MsgParsedAddressesCcHashed,
+                    MsgParsedAddressesCc,
+                    MsgHeaderCcHashed,
+                    MsgHeaderCc,
+                    FilterModulesAvVirusNames,
+                    FilterThrottleIp,
+                    FilterModulesUrldefenseCountsNoRewriteIsUnsupportedScheme,
+                    FilterModulesUrldefenseCountsNoRewriteIsSchemeless,
+                    FilterModulesUrldefenseCountsNoRewriteIsMaxLengthExceeded,
+                    FilterModulesUrldefenseCountsNoRewriteIsExcludedDomain,
+                    PpsAgent,
+                    SrcDvcHostname,
+                    EventOriginalTime,
+                    TlsCipher,
+                    TlsCipherBits,
+                    TlsVersion,
+                    SrcIpAddr,
+                    NetworkSessionId,
+                    SrcGeoCountry,
+                    NetworkProtocol,
+                    NetworkConnectionState,
+                    NetworkBytes,
+                    NetworkDuration,
+                  //  EventStartTime,
+                    NetworkDirection,
+                    DstUserUpn,
+                    SrcUserUpn,
+                    EventType
+        | join kind=inner  (
+    		ProofpointPODMessage_CL
+            | extend Guid = column_ifexists('guid', ''), MsgParts=column_ifexists('msgParts', '')
+    		    | extend d = parse_json(MsgParts)
+            | mv-expand d
+            | extend URL_LIST = parse_json(d.urls)
+            | mv-expand URL_LIST
+            | extend URLLIST = tostring(URL_LIST.url)
+    		    | summarize make_set(URLLIST) by Guid
+            )
+            on Guid
+        | project-away Guid1
+        | project-rename Urls = set_URLLIST
+    };
+    union isfuzzy=true ProofpointPOD_message_view, ProofpointPOD_maillog_view, ProofpointPOD_maillogV2_view, ProofpointPOD_messageV2_view
```

Comment:
did we test cases like when only v1 table is present and when only V2 table is present and when both tables are present?
What is the behavior?

Reply:
yes